### PR TITLE
Update to Debian-12 bookworm and back port changes from live.

### PR DESCRIPTION
**Pull request by tmcgilchrist**

### CODE REVIEW

File: lib/build.ml [hunk 1]
```diff
@@ -235,7 +236,7 @@ let list_revdeps t ~platform ~opam_version ~pkgopt ~base ~master ~after commit =
           | "" -> acc
           | revdep ->
               let revdep = OpamPackage.of_string revdep in
-              if OpamPackage.equal pkg revdep then
+              if List.exists (fun x -> OpamPackage.equal revdep x.PackageOpt.pkg) pkgs then
```
Comment (kit-ty-kate) [inline lib/build.ml:239]: @tmcgilchrist why was this included? Wasn't this a temporary commit? Why did you not keep the individual commits?

**Comment by tmcgilchrist** [comment]
I didn't realize it was a temp change. I can revert it tomorrow.

File: lib/build.ml (hunk 1 above)
Comment (tmcgilchrist) [reply lib/build.ml:239]: Revert PR https://github.com/ocurrent/opam-repo-ci/pull/229
